docs/source/API/core/initialize_finalize/InitializationSettings.rst

```diff
@@ -0,0 +1,87 @@
+``InitializationSettings``
+==========================
+
+.. role::cpp(code)
+    :language: cpp
+
+.. role:: cppkokkos(code)
+   :language: cppkokkos
+
+Defined in header ``<KokkosCore.cpp>``
+
+
+Usage:
+
+.. code-block:: cpp
+
+    auto settings = Kokkos::InitializationSettings()
+                    .set_num_threads(8)
+                    .set_device_id(0)
+                    .set_disable_warnings(false);
+
+``InitializationSettings`` is a class that can be used to define the settings for
+initializating Kokkos programmatically without having to call the two parameter
+form (``argc`` and ``argv``) of `Kokkos::initialize() <initialize.html#kokkosinitialize>`_.
+It was introduced in version 3.7 as a replacement for the
```

Comment:
Can you use `versionadded`? https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded

Reply:
fixed ,can you please check? @nmm0